### PR TITLE
Remove ordinal type

### PR DESCRIPTION
**Pull request by deboer-tim**
Someone mentioned this on Monday, but there is no reason we have a separate ordinal type - it's just a number with additional rules, but they are watered down enough to be meaningless. Removing the type and documenting within the two uses is simpler for adopters.

### CODE REVIEW

**Review by nickygerritsen — APPROVED** [review]

File: draft/Contest_API.md [hunk 1]
```diff
@@ -1383,7 +1376,7 @@ JSON elements of judgement objects:
 | id                   | ID      | yes       | no        | Identifier of the judgement.
 | submission\_id       | ID      | yes       | no        | Identifier of the [ submission](#submissions) judged.
 | judgement\_type\_id  | ID      | yes       | yes       | The [ verdict](#judgement-types) of this judgement.
-| score                | number  | no        | no        | Score for this judgement. Only relevant if contest:scoreboard_type is `score`. Defaults to `100` if missing.
```
Comment (niemela) [inline draft/Contest_API.md:1386]: Not related to ordinals.

File: draft/Contest_API.md [hunk 2]
```diff
@@ -1842,8 +1835,8 @@ Each problem object within the scoreboard consists of:
 | num\_judged  | integer | yes       | no        | Number of judged submissions (up to and including the first correct one),
 | num\_pending | integer | yes       | no        | Number of pending submissions (either queued or due to freeze).
 | solved       | boolean | depends   | yes       | Required iff contest:scoreboard_type is `pass-fail`.
-| score        | number  | depends   | no        | Required iff contest:scoreboard_type is `score` and solved is missing. If missing or `null` defaults to `100` if solved is `true` and `0` if solved is `false`.
```
Comment (niemela) [inline draft/Contest_API.md:1845]: Not related to ordinals.

**Review by niemela — CHANGES_REQUESTED** [review]
The ordinal change is good. :+1:

File: draft/Contest_API.md (hunk 1 above)
Comment (deboer-tim) [reply draft/Contest_API.md]: Fixed.

File: draft/Contest_API.md (hunk 2 above)
Comment (deboer-tim) [reply draft/Contest_API.md]: Fixed.

**Review by niemela — APPROVED** [review]

File: draft/Contest_API.md [hunk 3]
```diff
@@ -878,7 +870,7 @@ JSON elements of problem objects:
 | uuid              | string  | no        | yes       | UUID of the problem, as defined in the problem package.
 | label             | string  | yes       | no        | Label of the problem on the scoreboard, typically a single capitalized letter.
 | name              | string  | yes       | no        | Name of the problem.
-| ordinal           | ORDINAL | yes       | no        | Ordering of problems on the scoreboard.
+| ordinal           | number  | yes       | no        | A unique number useful only to order the problems, e.g. for the scoreboard.
```
Comment (eldering) [inline draft/Contest_API.md:873]: I would rephrase this as `A unique number that determines the order the problems, e.g. on the scoreboard.`

Comment (deboer-tim) [reply draft/Contest_API.md]: Agreed, updated.

**Review by eldering — APPROVED** [review]

**Review by nickygerritsen — APPROVED** [review]
